### PR TITLE
1.13

### PR DESCRIPTION
**Pull request by ngam**
Close #10 
Close #9 
Fix https://github.com/conda-forge/pytorch-cpu-feedstock/issues/148

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,65 +1,33 @@
...
-    - {{ compiler('cxx') }}
-    - python                                   # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}       # [build_platform != target_platform]
-    - pytorch                                  # [build_platform != target_platform]
-
   host:
-    - pip
-    - python
+    - python >=3.7
     - pytorch
```
Comment (hmaarrfk) [inline recipe/meta.yaml:18]:
```suggestion
    - pytorch {{ version }}
```

Comment (hmaarrfk) [reply recipe/meta.yaml:18]: Lets be explicit that it is just a dummy package.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,65 +1,33 @@
-{% set version = "0.2.1" %}
+{% set version = "1.13.0" %}
 
 package:
   name: functorch
   version: {{ version }}
 
 source:
   url: https://github.com/pytorch/functorch/archive/v{{ version }}.tar.gz
```
Comment (hmaarrfk) [inline recipe/meta.yaml:8]:
```suggestion
  # This is a dummy package and has been included into pytorch itself
  # https://github.com/pytorch/functorch/releases/tag/v1.13.0
  url: https://github.com/pytorch/functorch/archive/v{{ version }}.tar.gz
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,65 +1,33 @@
...
+  noarch: python
 
 requirements:
-  build:
-    - {{ compiler('cxx') }}
-    - python                                   # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}       # [build_platform != target_platform]
-    - pytorch                                  # [build_platform != target_platform]
-
   host:
```
Comment (awvwgk) [inline recipe/meta.yaml:16]: Do we actually have host dependencies?

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,65 +1,33 @@
...
-    - $RPATH/libc10_cuda.so
-    - $RPATH/libtorch_python.so
-    - $RPATH/libtorch_cpu.so
-    - $RPATH/libc10.so
-    - $RPATH/libc10_cuda.dylib
-    - $RPATH/libtorch_python.dylib
-    - $RPATH/libtorch_cpu.dylib
-    - $RPATH/libc10.dylib
+  number: 0
+  noarch: python
```
Comment (awvwgk) [inline recipe/meta.yaml:13]: Wouldn't that be `noarch: generic`?

**Review by awvwgk — APPROVED** [review]
Looks good to me. Only two minor comments / questions.

File: recipe/meta.yaml (hunk 4 above)
Comment (hmaarrfk) [reply recipe/meta.yaml:13]: maybe.... but lets leave it as a python package for now. I think you are right.

File: recipe/meta.yaml (hunk 3 above)
Comment (hmaarrfk) [reply recipe/meta.yaml:16]: i think you are generally correct, but lets just leave them here out of caution for a python package that doesn't exit.

Comment (awvwgk) [reply recipe/meta.yaml:16]: Then we need the same version pinning in host and run

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,65 +1,33 @@
...
-    - python
+    - python >=3.7
     - pytorch
   run:
-    - python
-    # we rely on global pinning to resolve pytorch
-    # functorch 0.1.x requires pytorch 1.11.x
-    # functorch 0.2.x requires pytorch 1.12.x
+    - python >=3.7
     - pytorch
```
Comment (hmaarrfk) [inline recipe/meta.yaml:21]:
```suggestion
    - pytorch {{ version }}
```

File: recipe/meta.yaml (hunk 3 above)
Comment (hmaarrfk) [reply recipe/meta.yaml:16]: yeah, ok lets do it.

**Comment by hmaarrfk** [comment]
Thank you for this!

**Pull request merged**